Core/MIPS/ARM64/Arm64CompFPU.cpp

```diff
@@ -111,7 +111,11 @@ void Arm64Jit::Comp_FPULS(MIPSOpcode op)
 			} else {
 				skips = SetScratch1ForSafeAddress(rs, offset, SCRATCH2);
 			}
-			MOVK(SCRATCH1_64, ((uint64_t)Memory::base) >> 32, SHIFT_32);
+			if (jo.enablePointerify) {
+				MOVK(SCRATCH1_64, ((uint64_t)Memory::base) >> 32, SHIFT_32);
+			} else {
+				ADD(SCRATCH1_64, SCRATCH1_64, MEMBASEREG);
```

Comment:
Note: Would be better to just use MEMBASEREG as the last arg to LDR and not do either of these?

-[Unknown]

Reply:
Well, that form of LDR takes an immediate offset, not a register, but changing to the register sum LDR would indeed probably be a win. Definitely not a loss. Don't remember, do we initialize MEMBASEREG even if pointerization is enabled?

Reply:
Though I suppose not needing to reload registers to "depointerize" should still be quite a win.